### PR TITLE
adding IPsec full offload settings to tests.

### PR DESCRIPTION
**Pull request by DorKovachi**

### CODE REVIEW

File: ngc_tcp_test.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@ SERVER_IP=$3
 SERVER_DEVICE=$4
 LOG_CLIENT=ngc_tcp_client_${CLIENT_IP}.log
 LOG_SERVER=ngc_tcp_server_${SERVER_IP}.log
+FIXED_MTU=$5
```
Comment (amirancel) [inline ngc_tcp_test.sh:19]: Add to usage as well

File: ngc_tcp_test.sh [hunk 2]
```diff
@@ -53,11 +54,15 @@ ssh ${SERVER_IP} systemctl stop irqbalance
 
 # Increase MTU to maximum per link type
 LINK_TYPE=`ssh ${CLIENT_IP} cat /sys/class/infiniband/${CLIENT_DEVICE}/device/net/\*/type`
+if [ ${FIXED_MTU} == fixed_mtu ]; then
+	MTU=1500
```
Comment (amirancel) [inline ngc_tcp_test.sh:58]: Verify that FIXED_MTU is not empty

Comment (amirancel) [inline ngc_tcp_test.sh:58]: Need to be the MTU Size not "fixed_mtu"

File: ngc_tcp_test.sh [hunk 3]
```diff
@@ -53,11 +54,15 @@ ssh ${SERVER_IP} systemctl stop irqbalance
...
+else
 if [ $LINK_TYPE -eq 1 ]; then
 	MTU=9000
 elif [ $LINK_TYPE -eq 32 ]; then
 	MTU=4092
 fi
+#fi
```
[the remark this answers is not in the document]
Comment (DorKovachi) [reply ngc_tcp_test.sh:65]: removed '#'
the "fi" was necessary because of nested if.

File: ngc_tcp_test.sh (hunk 2 above)
Comment (DorKovachi) [reply ngc_tcp_test.sh:58]: done.

File: ngc_tcp_test.sh (hunk 1 above)
Comment (DorKovachi) [reply ngc_tcp_test.sh:19]: done.

**Pull request closed without merging**